test/k8sT/DatapathConfiguration.go

```diff
@@ -280,6 +280,17 @@ var _ = Describe("K8sDatapathConfig", func() {
 			Expect(testPodHTTPToOutside(kubectl, "http://google.com", false, false)).
 				Should(BeTrue(), "Connectivity test to http://google.com failed")
 		})
+
+		It("Check iptables masquerading without random-fully", func() {
```

Comment:
Is it possible to move this `It()` under an existing `Context()` so that we can avoid having to redeploy Cilium again just for this test? If there's already a `Context()` that configures Cilium in the same way (BPF masq off), then we can do this optimization.

Reply:
I don't think it's possible as this is about testing a specific Cilium configuration (iptables masquerading without the random-fully option) which is not used in any other test